frontend/src/app/tasks/tasks.css

```
@@ -33,3 +33,42 @@
 .animate-slide {
   animation: slide-horizontal 1.5s ease-in-out infinite;
 }
+
+.chat-typography .wmde-markdown,
+.chat-typography .markdown-content,
+.chat-typography .enhanced-markdown,
+.chat-typography .smart-text-line,
+.chat-typography .chat-message-prompt {
+  font-family:
+    'Google Sans Flex', 'Google Sans', 'Noto Sans SC', 'Inter', 'PingFang SC', 'Noto Sans CJK SC',
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)